db/seeds.rb

```diff
@@ -1,7 +1,128 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Book.create!(
+  title: "Voice of War",
```

Comment:
`nit` 

You can use faker here to generate random book data. This is a very `nit` comment so feel free to disergard